docs/gettingstarted/example-data.rst

```diff
@@ -5,4 +5,6 @@ If you'd like some data to test your new storefront you can populate the databas
 
 .. code-block:: console
 
- $ python manage.py populatedb
+ $ python manage.py populatedb --createsuperuser
+
+The ``--createsuperuser`` switch creates an admin account for ``admin@example.com`` with the password set to ``admin``.
```

Comment:
I think it's usually called `argument` not `switch`

Reply:
I used the [docs](https://saleor.readthedocs.io/en/latest/customization/docker.html) as a reference, would you like me to change the ```switch``` there to ```argument``` as well?

Reply:
Yes please, I think the `argument` is more commonly used